boundary_layer_default_plugin/config/operators/gcp_cloud_function_invoke.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Etsy Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+# see: https://airflow.apache.org/docs/apache-airflow-providers-google/stable/_api/airflow/providers/google/cloud/operators/functions/index.html
+# and: https://github.com/apache/airflow/blob/master/airflow/providers/google/cloud/operators/functions.py
+
+name: gcp_cloud_function_invoke
+operator_class: CloudFunctionInvokeFunctionOperator
+operator_class_module: airflow.providers.google.cloud.operators.functions
+schema_extends: base
+parameters_jsonschema:
+    properties:
+        function_id:
+            type: string
+        input_data:
```

Comment:
`input_data` is Dict. I'm not sure if this is the right representation. PTAL.

Reply:
This is correct!